src/commonMain/rust/bridge/stream.rs

```diff
@@ -131,18 +131,32 @@ impl
             Option<&ResourceRequest>,
         ),
     ) -> types::Stream {
-        let streaming_server_url =
-            ctx.map(|ctx| ctx.profile.settings.streaming_server_url.to_owned());
-        let deep_links = if stream_request.is_some() && meta_request.is_some() {
-            StreamDeepLinks::from((
+        // in calls that have None for ctx this would panic if we don't set it to default.
+        let settings = ctx
+            .map(|ctx| ctx.profile.settings.to_owned())
+            .unwrap_or_default();
```

Comment:
- [ ] The streaming url should be taken from `StreamingServer::base_url` instead!

Reply:
@TheBeastLT @sleeyax any ideas where/how to best get the (model) `StreamingServer::base_url` and pass it to this function?

Reply:
Why would we need to take it from `StreamingServer`? Streaming server uses the url from settings, that's what was used here. Please revert this.

Reply:
Not always. If there's another running server (like starting Stremio service before desktop) you can have 2 different servers.
For consistency with web, we should strive for the same behavior in my opinion and this is where we get the url from for web:


https://github.com/Stremio/stremio-core-web/blob/c6dc920197bc233f4306f99d2fde27023d67d986/src/model/serialize_player.rs#L103-L107

But if you say consistency in this case doesn't matter that much we can leave it as it is. After all on androidtv and android we should see more than 1 instance of the server running, right?

Reply:
Web uses streaming server instance, as it needs to check if there's a server running. On android and android tv we don't need that as it depends on the server and expects it to be running.
But even if there are 2 server running, it doesn't matters, as like I said `StreamingServer` will use the url defined in settings, the 2nd running server has no impact.

Reply:
Ok, then we can leave it as it is.
I'm resolving this question as Not applicable.